eduid_am/config.py

```diff
@@ -1,113 +0,0 @@
-import os
-import os.path
-import ConfigParser
-
-
-DEFAULT_CONFIG_FILE_NAME = 'eduid_am.ini'
-
-
-def get_config_file():
-    """Get the configuration file looking for it in several places.
-
-    The lookup order is:
-    1. A file named acording to the value of the EDUIM_AM_CONFIG env variable
-    2. A file named eduid_am.ini In the current working directory
-    3. A file named .eduim_am.ini in the user's home directory
-    4. A file named eduim_am.ini in the system configuration directory
-    """
-    file_name = os.environ.get('EDUID_AM_CONFIG', DEFAULT_CONFIG_FILE_NAME)
-
-    if os.path.exists(file_name):
-        return file_name
-
-    user_file = os.path.expanduser(
-        os.path.join('~', '.', DEFAULT_CONFIG_FILE_NAME))
-    if os.path.exists(user_file):
-        return user_file
-
-    global_file = os.path.join('/etc', DEFAULT_CONFIG_FILE_NAME)
-    if os.path.exists(global_file):
-        return global_file
-
-def read_setting_from_env(settings, key, default=None):
-    env_variable = key.upper()
-    if env_variable in os.environ:
-        return os.environ[env_variable]
-    else:
-        return settings.get(key, default)
-
```

Comment:
The functions above and below are used outside  eduid_am, see for example [in the dashboard](https://github.com/SUNET/eduid-dashboard/blob/master/eduiddashboard/__init__.py#L16)